docs/en/basic/Command-Line-Interface.md

```diff
@@ -155,6 +166,11 @@ And the following command returns the log level of the class `alluxio.heartbeat.
 alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
 ```
 
+### runClass
```

Comment:
This is developer facing command that will be better to put in `Contributor-tools.md`

Reply:
done